### PR TITLE
hello-world example

### PR DESCRIPTION
**Pull request by drmingdrmer**
This change is [Reviewable](https://reviewable.io/reviews/suraft/suraft/6)

### CODE REVIEW

File: examples/hello/src/logging.rs [hunk 1]
```diff
@@ -0,0 +1,114 @@
+use std::backtrace::Backtrace;
+use std::env;
+use std::panic::PanicHookInfo;
+
+use tracing::Event;
```
Comment (tisonkun) [inline examples/hello/src/logging.rs:5]: May we use log + logforth (+fastrace)?

Comment (tisonkun) [reply examples/hello/src/logging.rs:5]: Seems we are using tracing everywhere inside suraft. Perhaps revisit this topic later.

**Review by tisonkun — APPROVED** [review]
Generally looks good. I'd expect give an overall review once the first version (0.1.0) (or its release candidate) is ready.

It seems inherit a lot from OpenRaft's design and I need some time to check all of them.

File: examples/hello/hello-suraft-cn.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
+# 使用 suRaft 构建简单的 3 节点集群
+
+本教程将引导你一步步使用 **suRaft** 来实现一个简单的 3 节点集群。
+这个集群允许写入用Cmd定义的用户数据, 将其持久化到共享存储中,
```
[the remark this answers is not in the document]
Comment (drmingdrmer) [reply examples/hello/hello-suraft-cn.md:4]: Hmm... I never imagined there would be guidelines for this style. Nice guideline and plugins!

**Pull request closed without merging**